spec/models/base_organisation_spec.rb

```diff
@@ -56,5 +56,37 @@
       org.save
     end
   end
+  
+  describe 'friendly_id' do
+    # TODO: more tests on friendly_id methods?
```

Comment:
maybe if we move these to setupslug_spec and ensure the coverage is maintained then we should be good to go